lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -5821,12 +6858,73 @@ def gather_inputs_parametric_schedules(sch, load_inst, parametric_inputs, hours_
     return parametric_inputs
   end
 
+  # Retrieves the lowest story in a model
+  #
+  # @param model [OpenStudio::model::Model] OpenStudio model object
+  # @return [OpenStudio::model::BuildingStory] Lowest story included in the model
+  def find_lowest_story(model)
+    min_z_story = 1E+10
+    lowest_story = nil
+    model.getSpaces.sort.each do |space|
+      story = space.buildingStory.get
+      lowest_story = story if lowest_story.nil?
+      space_min_z = building_story_minimum_z_value(story)
+      if space_min_z < min_z_story
+        min_z_story = space_min_z
+        lowest_story = story
+      end
+    end
+    return lowest_story
+  end
+
+  # Utility function that returns the min and max value in a design day schedule.
+  #
+  # @param schedule [OpenStudio::Model::Schedule] can be ScheduleCompact, ScheduleRuleset, ScheduleConstant
+  # @param type [String] 'heating' for winter design day, 'cooling' for summer design day
+  # @return [Hash] Hash has two keys, min and max. if failed, return 999.9 for min and max.
+  def search_min_max_value_from_design_day_schedule(schedule, type = 'winter')
```

Comment:
move to a Standards.Schedule file

Reply:
I've added a TODO, to take care of this next year.